server/views/case-summary-record.njk

```diff
@@ -91,17 +100,7 @@
             {% endif %}
 
         </div>
-        <div class="govuk-grid-column-one-third">
```

Comment:
This needs to still be here but only display for current offenders.

Reply:
cool